stackalytics/html_dumper.py

```diff
@@ -55,13 +42,17 @@ def collect_activities_per_user(self, user_id, release):
                 break
         return activities
 
-    def commits(self):
+    def collect_commit_info(self, releases=['all']):
+        """
+        :param releases: which release caller wants to collect
+        :return: a dict which maps release -> user_id -> commit info
+        """
         commits = {}
         for release in releases:
             commits_per_release = {}
             for user_id in self.user_id_list:
-                activities_per_user_and_release =\
-                    self.collect_activities_per_user(user_id, release)
+                activities_per_user_and_release = self.collect_activities(
+                    user_id, release, self.filter_params)
```

Comment:
self.filter_params is not set in BaseCollector, so I think this variable should be set in subclass.
But in LoCCollector, the variable is not set. Is this supposed to be this way?